### PR TITLE
feature/add-pagination: Add pagination on fetching movies and users data

### PR DESCRIPTION
**Pull request by Abhigyandhakallf**
## Describe your changes
Add pagination on fetching movies and users data

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.
- [ ] I have created postman test scripts for it

## Postman Screenshots
Movies:
image

Users:
image

## Postman pull request URL
`http://localhost:8848/api/users?page=1&pageSize=10`

`http://localhost:8848/api/movies?page=2&pageSize=10`

### CODE REVIEW

File: src/controllers/usersController.js [hunk 1]
```diff
@@ -10,8 +10,10 @@ import * as userService from '@services/userService';
  * @param {Function} next
  */
 export function fetchAll(req, res, next) {
+  const { page = 1, pageSize = 10 } = req.query;
```
Comment (ayxsth) [inline src/controllers/usersController.js]: Let's store the default values to a constant.

**Review by ayxsth — APPROVED** [review]

**Comment by ayxsth** [comment]
Need to update the test cases as well.

**Review by sbimochan — APPROVED** [review]

**Comment by sbimochan** [comment]
would be great to see few pagination assertions in postman 😄

**Comment by sbimochan** [comment]
No worries, i can contribute that

**Pull request merged**